### DISCUSSION TITLE
How to open a support ticket

### DISCUSSION THREAD

**Discussion by ran-codes** [community/community#46983] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I am using a PRO account and I am trying to submit a Ticket via the GitHub Support Portal as per these [instructions](https://docs.github.com/en/support/contacting-github-support/creating-a-support-ticket).

However when I click the `New Ticket` button (see snip 1) it just brings me back to the support home page. Is this a bug or am I doing something wrong.

### Snip 1

![image](https://user-images.githubusercontent.com/45013044/218130797-9d869aca-4f2e-48ef-b635-d2ec760e4399.png)

### Snip 2
![image](https://user-images.githubusercontent.com/45013044/218130856-46e2bce1-f6b4-4759-a644-76840de45161.png)

**Comment by ran-codes** [comment]
Well. I saw in another post someone linked to https://support.github.com/contact/account.

THis allows me to submit a ticket... but still the `new issue` button bug above is still confusing.

**Comment by SilasRodrigues19** [comment, marked as answer]
Hey @ran-codes 

Click on this link here below, you should see a screen as shown in the screenshot:

https://support.github.com/contact/account

![image](https://user-images.githubusercontent.com/47532703/218139831-00b0edeb-532d-4d2c-9ce2-245c8d395af4.png)

**Reply by ran-codes** [reply]
Thanks!! that works.

**Reply by SilasRodrigues19** [reply]
But I saw here that you are right, the button is pointing to `/contact` but for some reason when we click it redirects
to `/request`.

https://user-images.githubusercontent.com/47532703/218141417-63582cc3-7b9d-48a9-ba8f-53d6c560cb4b.mp4

**Reply by ran-codes** [reply]
right!? 😦

**Comment by BobPChand** [comment]
Hi, I somehow deleted the account. I need to retrieve this, please. curl -L \
  -X DELETE \
  -H "Accept: application/vnd.github+json" \
  -H "Authorization: Bearer <YOUR-TOKEN>" \
  -H "X-GitHub-Api-Version: 2022-11-28" \
  https://api.github.com/repos/OWNER/REPO/releases/assets/ASSET_ID

**Comment by BobPChand** [comment]
I think I made a mistake again. Please add it back to the site with many thanks
